article/app/model/structuredData/Image.scala

```diff
@@ -1,16 +1,19 @@
 package model.structuredData
 
-import model.ImageElement
-import play.api.libs.json.{JsValue, Json}
-import views.support.{ImgSrc, Item700}
+import model.{ImageAsset, ImageElement}
+import play.api.libs.json.{JsString, JsValue, Json}
+import views.support.Item700
 
 object Image {
 
-  def apply(picture: ImageElement): JsValue = Json.obj(
-    "@type" -> "ImageObject",
-    "url" -> ImgSrc.findNearestSrc(picture.images, Item700),
-    "height" -> ImgSrc.getFallbackAsset(picture.images).fold(0)(_.height),
-    "width" -> ImgSrc.getFallbackAsset(picture.images).fold(0)(_.width)
-  )
+  def apply(picture: ImageElement): JsValue = {
+    val asset: Option[ImageAsset] = Item700.bestFor(picture.images)
+    Json.obj(
+      "@type" -> "ImageObject",
+      "url" -> JsString(Item700.bestSrcFor(picture.images).getOrElse("")),
+      "height" -> asset.fold(0)(_.height),
```

Comment:
Technically this isn't the true width even - it will just be `Item700.width` when imgix in use.